src/Tempest/Cache/src/IsCache.php

```diff
@@ -36,6 +38,10 @@ public function get(string $key): mixed
     /** @param Closure(): mixed $cache */
     public function resolve(string $key, Closure $cache, ?DateTimeInterface $expiresAt = null): mixed
     {
+        if (! $this->isEnabled()) {
```

Comment:
This `isEnabled` check only happens when calling resolve, we'll need to decide whether it should also be checked in other cache methods (out of scope for this PR)